libs/datalayout/src/main/java/com/linkedin/openhouse/datalayout/datasource/PartitionStat.java

```diff
@@ -0,0 +1,18 @@
+package com.linkedin.openhouse.datalayout.datasource;
+
+import java.io.Serializable;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/** Represents table partition. */
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class PartitionStat implements Serializable {
+  private List<String> values;
```

Comment:
what does values signify? Add comment and try different variable name.

Reply:
Added comment